include/grpcpp/impl/codegen/server_context_impl.h

```diff
@@ -88,36 +95,29 @@ namespace grpc {
 class GenericServerContext;
 class ServerInterface;
 
+namespace experimental {
+class GenericCallbackServerContext;
+}  // namespace experimental
+
 namespace internal {
 class Call;
 }  // namespace internal
 
 namespace testing {
 class InteropServerContextInspector;
 class ServerContextTestSpouse;
+class DefaultReactorTestPeer;
```

Comment:
https://abseil.io/tips/135 consistently calls it a Peer and says that it's sometimes called a spouse. (This was earlier in an internal ToTW 135).  We can't change the name of the Spouse now (to avoid breaking existing users) but we shouldn't add deprecated terms.

Reply:
I don't see anything in there that suggests Spouse is deprecated.

It would be nice if we could be internally consistent, but I honestly prefer the term "peer" over "spouse", so I guess this is fine.